react-catalog/my-app/src/components/Header.tsx

```diff
@@ -0,0 +1,26 @@
+import { Outlet } from 'react-router';
+import { Link } from 'react-router-dom';
+
+export function Header() {
+  return (
+    <>
+      <header className="flex items-center h-12 bg-slate-800">
+        <nav>
+          <ul className="flex gap-6 pl-6">
+            <li>
+              <Link to={'/about'} className="text-white">
```

Comment:
You only need the brackets if you're embedding a JavaScript expression. Here, you could just do `to="/about"`